modules/scheduler-impl/src/test/java/org/opencastproject/scheduler/impl/SchedulerServiceImplTest.java

```diff
@@ -626,18 +597,20 @@ public void nonExistantRecording() throws Exception {
       assertNotNull(e);
     }
 
+    /*
```

Comment:
@lkiesow That is truth is that I haven't yet managed to get this test working... (TBD)